resources/views/includes/status.blade.php

```diff
@@ -55,7 +55,7 @@
                 </li>
                 <li>
```

Comment:
<img width="1370" alt="Screenshot 2021-04-17 at 11 20 02" src="https://user-images.githubusercontent.com/4103693/115108086-dc873700-9f6e-11eb-851f-542624c1edbd.png">
The status cards on the user profile are in the "old" size and thus no longer centered on the page.